plugins/woocommerce-blocks/assets/js/blocks/cart/block.js

```diff
@@ -26,11 +26,19 @@ import './style.scss';
 const reloadPage = () => void window.location.reload( true );
 
 const Cart = ( { children, attributes = {} } ) => {
-	const { cartIsLoading } = useStoreCart();
+	const cart = useStoreCart();
 	const { hasDarkControls } = attributes;
 
+	const { dispatchStoreEvent } = useStoreEvents();
+
+	// Ignore changes to dispatchStoreEvent callback so this is ran on first mount only.
+	useEffect( () => {
+		dispatchStoreEvent( 'cart-render', { cart } );
```

Comment:
Can be done in a follow up maybe, but doing the same event to Checkout would be useful.

Reply:
There is already a similar event for checkout here: https://github.com/woocommerce/woocommerce/blob/14cd6e6dfd7459d462aad889911e9a5a93ad623d/plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-fields-block/frontend.tsx#L21